packages/website/custom-plugins/mdx-image-loader.js

```diff
@@ -9,43 +9,44 @@ module.exports = () => tree =>
   visit(tree, 'root', rNode => {
     visit(tree, 'image', node => {
       const text = node.url ? node.url.trim() : '';
+      if (text.startsWith('./') || text.startsWith('../')) {
```

Comment:
I've noticed we're using `startsWith` in a couple of places, do we ship any polyfills for browsers that don't support this (IE11)?

Reply:
We don't - I think we don't target ie11.

But also, this is node-only code so here it won't cause us any grief.